emission/integrationTests/suggestionsys/find_candidate_business.dataset.json

```diff
@@ -0,0 +1,435 @@
+[
+    {
+        "test_name": "Mountain View Kohls",
+        "test_type": "Suburban big box store in local anchor shopping center ",
+        "input": {
+            "loc": {
+                "coordinates": [-122.1078300, 37.4035500],
+                "type": "Point"
+            }
+        },
+        "output": ["Kohl's Mountain View", "350 Showers Drive", "Mountain View", true]
```

Comment:
what is this dataset and how is it different from `find_destination_business.dataset.json`

Reply:
Sorry, this was part of the version that was not up to date with the recent fetch to the branch. It is deleted now.